### DISCUSSION TITLE
Bug: `vercel dev` is using encrypted version of env vars

### DISCUSSION THREAD

**Discussion by websocket98765** [vercel/vercel#5459] [Help] [answered]
For some reason, today using `vercel dev` loaded my env vars in what appears to be the still-encrypted version--e.g. `sec_dj284DHuhsdfih`.

If I run `vercel env pull` to confirm, it creates a .env file with all my env vars having these encrypted values (`sec_dj284DHuhsdfih`, etc) and I can't see plaintext versions of my env vars, as previously possible.

I upgraded vercel CLI to v21.0.0 and the issue remained, so I believe it's platform-related, rather than CLI related.

Vercel CLI: 21.0.0

**Comment by mcsdevv** [comment, marked as answer]
Hello @websocket98765, sorry about this, we'll get this fixed shortly.

For now, you can use the 19.2.0 CLI version to workaround the issue.

**Reply by websocket98765** [reply]
Thank you. v19.2.0 worked!
